packages/pkgpanda-api/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p $PKG_PATH/etc/
+
+cat <<EOF > $PKG_PATH/etc/pkgpanda-api.conf.py
```

Comment:
It lets `pkgpanda.http.app.config['WORK_DIR']` default to a location under `/tmp`, which is a little friendlier for local hacking than `/run`.